src/main/java/com/fryrank/validator/package-info.java

```diff
@@ -0,0 +1,4 @@
+@NonNullApi
+package com.fryrank.validator;
+
+import org.springframework.lang.NonNullApi;
```

Comment:
We can remove this

Reply:
Done in [c52da6f](https://github.com/NickPriv/FryRankBackend/pull/62/commits/c52da6f116a93053a61ae696f15c2504a2a2a5c4).